docs/project/faq.md

```diff
@@ -310,6 +314,99 @@ actively encourage using these languages in those cases. However, we need a
 solution for C++ use-cases that require its full performance, low-level control,
 and access to hardware.
 
+## Why is Carbon designed that way?
+
+### Why aren't `<` and `>` used as delimiters?
+
+[One of our goals for Carbon](/docs/project/goals.md#fast-and-scalable-development)
+is that it should support parsing without contextual or semantic information,
+and experience with C++ has shown that using `<` as both a binary operator and
+an opening delimiter makes that goal difficult to achieve.
+
+For example, in C++, the expression `a<b>(c)` could parse as either a function
+call with a template argument `b` and an ordinary argument `c`, or as a chained
+comparison `(a < b) > (c)`. In order to resolve the ambiguity, the compiler has
+to perform name lookup on `a` to determine whether there's a function named `a`
+in scope.
+
+It's also worth noting that Carbon doesn't use _any_ kind of brackets to mark
+template parameters (or generic parameters), so if Carbon had angle brackets,
```

Comment:
```suggestion
[template parameters (and also generic parameters)](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/README.md#checked-and-template-parameters), so if Carbon had angle brackets,
```

To my below comment, which I've marked resolved, I do think there should be some link/reference for what _is_ used for template parameters. But maybe this reflects your intent?

(also, suggesting replacing "or" with "and also" to make clear to newcomers that "template" and "generic" aren't two terms for the same thing)